app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    operator, existing_file, new_file = command.split(" ")
```

Comment:
Ensure that the input string contains exactly three elements separated by spaces and starts with 'cp'. This is a requirement from the checklist to validate the input properly.